File: adapters/visited-places/src/main/java/sk/slsp/workshop/mazesolver/adapters/visitedplaces/configuration/VisitedPlacesAdapterConfiguration.java
package sk.slsp.workshop.mazesolver.adapters.visitedplaces.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import sk.slsp.workshop.mazesolver.adapters.visitedplaces.adapter.VisitedPlacesInMemoryAdapter;

@Configuration
public class VisitedPlacesAdapterConfiguration {
	
	@Bean
	public VisitedPlacesInMemoryAdapter visitedPlacesInMemoryAdapter() {
		return new VisitedPlacesInMemoryAdapter();
	}
	
}

File: adapters/visited-places/src/main/kotlin/sk/slsp/workshop/mazesolver/adapters/visitedplaces/configuration/VisitedPlacesAdapterConfiguration.kt
package sk.slsp.workshop.mazesolver.adapters.visitedplaces.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import sk.slsp.workshop.mazesolver.adapters.visitedplaces.adapter.VisitedPlacesInMemoryAdapter

@Configuration
class VisitedPlacesAdapterConfiguration {

	@Bean
	fun visitedPointsInMemoryAdapter()
		= VisitedPlacesInMemoryAdapter()

}